### PR TITLE
Clean up customer subscription code

### PR DESCRIPTION
**Pull request by taers232c**

### CODE REVIEW

File: src/gam.py [hunk 1]
```diff
@@ -8330,26 +8330,23 @@ def doWhatIs():
...
-  results = callGAPI(res.subscriptions(), u'list', customerId=customerId)
-  for subscription in results[u'subscriptions']:
-    if sku == subscription[u'skuId']:
-      subscriptionId = subscription[u'subscriptionId']
-      break
-  if not subscriptionId:
-    print u'ERROR: could not find subscription for customer %s and SKU %s' % (customerId, sku)
-    sys.exit(3)
-  return subscriptionId
+def getCustomerSubscription(res):
```
Comment (jay0lee) [inline src/gam.py]: What is the advantage of these changes?

**Comment by taers232c** [comment]
doDeleteResoldSubscription and doUpdateResoldSubscription take the same three arguments so I make s subroutine that gets them as a unit.

**Comment by taers232c** [comment]
I'll put it back your way

File: src/gam.py [hunk 2]
```diff
@@ -8330,26 +8330,23 @@ def doWhatIs():
...
+  customerId = sys.argv[3]
+  skuId = sys.argv[4]
```
Comment (jay0lee) [inline src/gam.py]: I don't like these utility functions depending on sys.argv order, it means they can't be used later if argument order is different. I think convertSKU2ProductId() is pretty much fine as is though the argument cleanup in _getResoldCustomerAttr() is good.

**Pull request closed without merging**